Sparkle/SUAppcastItem.m

```diff
@@ -55,6 +55,11 @@ - (BOOL)isCriticalUpdate
     return [self.propertiesDictionary[SUAppcastElementTags] containsObject:SUAppcastElementCriticalUpdate];
 }
 
+- (BOOL)isInformationOnlyUpdate
+{
+	return (([self infoURL] != nil) && ([self fileURL] == nil));
```

Comment:
Just to add: in a situation like `if (!number)`, you can't tell offhand if it's a NSNumber object, int*, or int. If you explicitly compare to != nil, you know offhand. If you change the type of the variable during refactoring, a C compiler can warn you that you need to change this code. I come from the standpoint where implicit testing should only be used for "boolean" (not pointers or integers) types. Languages like haskell and java enforce this. So, IMO, I find explicit testing more readable.